packages/mobile/src/firebase/saga.ts

```diff
@@ -100,55 +100,62 @@ function celoGoldExchangeRateHistoryChannel(lastTimeUpdated: number) {
   }
 
   const now = Date.now()
+  // timestamp + 1 is used because .startAt is inclusive
+  const startAt = Math.max(lastTimeUpdated + 1, now - MAX_HISTORY_RETENTION)
 
   return eventChannel((emit: any) => {
-    const emitter = (snapshot: FirebaseDatabaseTypes.DataSnapshot) => {
+    const singleItemEmitter = (snapshot: FirebaseDatabaseTypes.DataSnapshot) => {
```

Comment:
Can we define these outside of the `eventChannel`? It makes it a little more difficult to follow when all these functions are nested

Reply:
We can't because it uses the `emit` thing that is returned in the `eventChannel` :/